clients/src/main/java/org/apache/kafka/common/requests/DescribeConfigsResponse.java

```diff
@@ -293,158 +208,66 @@ public ConfigSource source() {
         }
     }
 
+    public Map<ConfigResource, DescribeConfigsResponseData.DescribeConfigsResult> resultMap() {
+        return data().results().stream().collect(Collectors.toMap(
+            configsResult ->
+                    new ConfigResource(ConfigResource.Type.forId(configsResult.resourceType()),
+                            configsResult.resourceName()),
+            Function.identity()));
+    }
 
-    private final int throttleTimeMs;
-    private final Map<ConfigResource, Config> configs;
+    private final DescribeConfigsResponseData data;
 
-    public DescribeConfigsResponse(int throttleTimeMs, Map<ConfigResource, Config> configs) {
-        this.throttleTimeMs = throttleTimeMs;
-        this.configs = Objects.requireNonNull(configs, "configs");
```

Comment:
Hey @mimaison @tombentley, removing this constructor seems to have broken the build (it was used in `TopicAdminTest`). Can we get a hotfix for this to unblock other PRs?